v2/api-validator/tests/server-tests/collateral/account-link.test.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  CollateralAccount,
+  CollateralAccountLink,
+  CollateralLinkStatus,
+  CollateralSignerId,
+  CryptocurrencyReference,
+  Environment,
+} from '../../../src/client/generated';
+import { getCapableAccountId } from '../../utils/capable-accounts';
+import { Pageable, paginated } from '../../utils/pagination';
+import config from '../../../src/config';
+import Client from '../../../src/client';
+
+describe('Account Link', () => {
+  const client = new Client();
+
+  describe('Creates a new link between a collateral account and a provider account', () => {
+    describe.each([
+      { env: Environment.PROD, expectedTestAsset: false },
+      { env: Environment.SANDBOX, expectedTestAsset: true },
+    ])('Successful request', (testParams) => {
+      const { env, expectedTestAsset } = testParams;
+      it('Should return valid response', async () => {
+        const accountId: string = getCapableAccountId('collateral');
+        const collateralId: string = config.get('collateral.collateralAccount.accountId');
+        const collateralSigners: CollateralSignerId[] = config.get('collateral.signers.userId');
+        const response: CollateralAccountLink = await client.collateral.createCollateralAccountLink(
+          {
+            accountId,
+            requestBody: {
+              collateralId,
+              collateralSigners,
+              env: env,
+            },
+          }
+        );
+
+        expect(response.collateralId).toBe(collateralId);
+        expect(response.collateralSigners).toEqual(collateralSigners);
+        expect(response.rejectionReason).toBeUndefined();
+        expect(response.status).toBe(CollateralLinkStatus.ELIGIBLE);
+        expect(response.env).toBe(env);
+
+        const assetsList: CryptocurrencyReference[] = response.eligibleCollateralAssets;
+        for (const asset of assetsList) {
+          if (typeof asset['testAsset'] === 'boolean') {
+            expect(asset['testAsset']).toBe(expectedTestAsset);
```

Comment:
This assumption is not true. Just remove it.

Reply:
removed